metadata/RNA-Seq.yml

```diff
@@ -56,6 +56,42 @@ material:
         type: "internal"
         link: "/Introduction/IGV"
 
+-
+    title: "De novo transcriptome reconstruction with RNA-seq"
+    type: "tutorial"
+    name: "de_novo"
+    zenodo_link: "https://zenodo.org/record/254485#.WKODmRIrKRu"
+    galaxy_tour: "..."
+    hands_on: "yes"
+    slides: "no"
+    
+    questions:
+    - "What genes are differentially expressed between G1E cells and megakaryocytes?"
+    - "How can we generate a transcriptome de novo from RNA sequencing data?"
+    
+    objectives:
+    - "Analysis of RNA sequencing data using a reference genome"
+    - "Reconstruction of transcripts without reference transcriptome (de novo)"
+    - "Analysis of differentially expressed genes"
+
+    time_estimation: "1d"
+
+    key_points:
+    - "Simple sentence to sum up the first key point of the tutorial (Take home message)"
+    - "Second key point"
+    - "Third key point"
+    - "..."
+
+    requirements:
+    -
+        title: "Mapping"
+        type: "internal"
+        link: "/NGS-mapping"
+    -
+        title: "IGV"
```

Comment:
Trackster is a requirement as well, right? I have the feeling that not many people have a fully functional trackster these days. So maybe we should mention this.